migrations/sql/beforeMigrate__0__add_extensions.sql

```diff
@@ -0,0 +1,3 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
```

Comment:
@FreshGyan  knows this but for other MDS reviewers, this is simply a 'failsafe' as our image already has pgcrypto and postgis enabled.